### PR TITLE
Add: Community wallpapers

### PR DESCRIPTION
**Pull request by Owen-sz**
Part 1 of community wallpapers package.

### CODE REVIEW

File: gnome/41-community-extras.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        <author>Inderix</author>
+    </wallpaper>
+    <wallpaper deleted="false">
+        <name>Lazuli</name>
+        <filename>/usr/share/backgrounds/ultramarine-linux/41-community-extras/Lazuli-light.png</filename>
+        <filename-dark>/usr/share/backgrounds/ultramarine-linux/41-community-extras/Lazuli-dark.png</filename-dark>
+        <options>zoom</options>
+        <author>Inderix</author>
+    </wallpaper>
+        <wallpaper deleted="false">
```
Comment (madonuko) [inline gnome/41-community-extras.xml]: indent

File: gnome/41-community-extras.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        <name>Lumos Dawn</name>
+        <filename>/usr/share/backgrounds/ultramarine-linux/41-community-extras/Lumos-Dawn.PNG</filename>
+        <options>zoom</options>
+        <author>Artemis</author>
+    </wallpaper>
+    <wallpaper deleted="false">
+        <name>Really Long Name</name>
```
Comment (madonuko) [inline gnome/41-community-extras.xml:25]: is this really called "Really Long Name"?

**Review by madonuko — COMMENTED** [review]
please also add the corresponding KDE folders

File: gnome/41-community-extras.xml (hunk 2 above)
Comment (Owen-sz) [reply gnome/41-community-extras.xml:25]: Yes

**Review by madonuko — APPROVED** [review]

**Pull request merged**